tests/conftest.py

```diff
@@ -9,6 +9,19 @@
 import numpy
 import pytest
 import yaml
+
+# Import pkl_* fixtures for tests in test_storage_resource
+# Need to also import parent fixtures to make fixtures work
+from functional.commands.conftest import (
```

Comment:
I don't understand why you need to import these here. They should only be available in the scope of `tests/functional/commands`. This makes them globally available.

Reply:
Ok ! Imports moved into `tests/functional/conftest` instead.